vuepress/docs/next/tutorials/jhipster/jhipster.md

```diff
@@ -0,0 +1,149 @@
+     - `Do you want pagination on your entity?` (Yes, with infinite scroll)
+     - At this point the blueprint will generate controllers, repositories, services, and micro frontends for your entity generation.
+
+ 7. When prompted with a conflict at this stage enter `a` for All.
+
+At this point you have a choice. Continue to follow this tutorial to move forward to build your Entando bundle and deploy your microservice and micro frontends to the Entando Component Repository. Or go to the [Running Locally](./run-local.md) tutorial to run your micro frontends and microservice in your local dev environment.
+
+## Build and Deploy
+This section will walk you through building a docker image from your microservice, creating your Entando bundle, checking your bundle into git, and deploying it to the Entando Component Repository.
+
+You'll need:
+  - Docker
+  - A running Entando instance (see [Getting started](../../docs/getting-started) for steps if needed)
+  - A bash shell
+  - git
+  - An empty git repository
+
+### Build Docker Image for Microservices
+1. In your microservice project on a command line run `./mvnw -Pprod clean package jib:dockerBuild`
+
+> **Note**
+>
+> By default the organization used to generate the docker image is `entando`, but you can provide a custom value during project initialization, as well as by changing the `pom.xml` file or by providing the `-Djib.to.image=<org>/<name>:<version>` to the `jib:dockerBuild` command.
+
+> **Note**
+>
+> Output image name is generated using the organization value defined during project initialization. You can override the provided values by altering the `pom.xml` file or by customizing the `-Djib.to.image` parameter used in the `./mvnw` command
+
+> **Warning**
+>
+> If you manually override the target image of the docker build, remember to update the plugin metadata in the bundle accordingly in the bundle steps.
+
+2. View your image and tag with `docker images`
+3. Publish the Docker image to Docker registry (DockerHub or equivalent `docker push <name-of-the-image:tag>`
+  > **Note**
+  >
+  > The first time your run this command it will have to push all of the layers. Subsequent runs will be much faster
+
+### Build your Bundle and publish to git
+1. In your microservice project populate the bundle with the generated micro frontends, run the `./buildBundle.sh` script or use `npm run populate-bundle`
+
+> **Important**
+> The bundle population with the micro frontends requires some time to be processed. You should be able to follow the progress of the operation on screen.
+
+2. The output of your bundle will be in the top level `bundle` folder in your microservice
+
+3. Create a new git repository and name it `my-bundle` (or a name of your choice)
+
+4. Clone the repository created in the prior step to a location of your choice (not inside your microservice) `git clone <url of your repo>`
+
+5. Copy the contents of the `bundle/` folder from your microservice into your repository
```

Comment:
Doesn't this make maintenance more difficult since there's a manual step to move the files from bundle/ over to the other repo? The existing instructions used a .gitignore and git init in the bundle/ directory to point it to a different repo. Also a bit awkward so I wonder if what you have here with a symlink to the controlled location might be better maintenance-wise.